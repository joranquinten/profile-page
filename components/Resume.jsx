import React from 'react';
import { Tabs } from 'antd';

import styled from 'styled-components';
import { colors } from '../styled/themes/default';

import Werkervaring from './Werkervaring';
import Projecten from './Projecten';
import Kennis from './Kennis';

const { TabPane } = Tabs;

const StyledResume = styled.div`
  .ant-tabs-nav-container .ant-tabs-nav .ant-tabs-tab-active {
    color: ${colors.primaryColor};
  }
  .ant-tabs-nav-container .ant-tabs-ink-bar {
    background-color: ${colors.primaryColor};
  }
  .ant-tabs-nav-container .ant-tabs-nav {
    color: ${colors.accentColor};
  }
`;

const Resume = () => (
  <StyledResume>
    <Tabs defaultActiveKey="1">
      <TabPane tab="Werkervaring" key="1">
        <Werkervaring />
      </TabPane>
      <TabPane tab="Projecten" key="2">
        <Projecten />
      </TabPane>
      <TabPane tab="Kennis" key="3">
        <Kennis />
      </TabPane>
    </Tabs>
  </StyledResume>
);

export default Resume;
