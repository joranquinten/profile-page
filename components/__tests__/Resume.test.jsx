import { shallow } from 'enzyme';

import Resume from '../Resume';

jest.mock('../../data/nl.js');

describe('<Resume />', () => {
  it('should render without crashing', () => {
    expect(shallow(<Resume />)).toMatchSnapshot();
  });
});
