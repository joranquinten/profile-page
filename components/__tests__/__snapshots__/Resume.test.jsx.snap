// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Resume /> should render without crashing 1`] = `
<Resume__StyledResume>
  <Tabs
    defaultActiveKey="1"
    hideAdd={false}
    tabPosition="top"
  >
    <TabPane
      key="1"
      placeholder={null}
      tab="Werkervaring"
    >
      <Werkervaring />
    </TabPane>
    <TabPane
      key="2"
      placeholder={null}
      tab="Projecten"
    >
      <Projecten />
    </TabPane>
    <TabPane
      key="3"
      placeholder={null}
      tab="Kennis"
    >
      <Kennis />
    </TabPane>
  </Tabs>
</Resume__StyledResume>
`;
